### ISSUE TITLE
hljs.COMMENT is not a function

### ISSUE THREAD

**Issue by MariuSteinbach** [highlightjs/highlightjs-cshtml-razor#1]
I'm using
```
<link rel="stylesheet" href="/Resources/Shared/stylesheets/vs2015.css">
<script src="/Resources/Shared/scripts/highlight.pack.js"></script>
<script type="text/javascript" src="/Resources/Shared/scripts/cshtml.js"></script>
<script>
  hljs.registerLanguage('cshtml', window.hljsDefineRazorCshtml);
  hljs.initHighlightingOnLoad();
</script>
```
and getting this error
```
Uncaught TypeError: hljs.COMMENT is not a function
    at hljsDefineRazorCshtml (cshtml.js:373)
    at Object.N [as registerLanguage] (highlight.pack.js:2)
    at adding-a-site:13
```

I'm using hljs v9.14.1 and the current cshtml.js

**Comment by aaronpowell** [comment]
I've just hit it myself, it's likely because the minified version of highlightjs that you have loaded has had the `COMMENT` function renamed.

I dug through the minified source and (for me) it looks like the function was renamed to `C`, so I shim it with `hljs.COMMENT = hljs.C` before I do the registration.

**Comment by romanresh** [comment]
Hi @aaronpowell, @MariuSteinbach . The issue was caused by the built-in minification system implemented in the original highlightjs project. It worked well while all languages were built-in in the only main project and their source code was minified with the source of hljs core. But this approach is not compatible with external language packages (like this one). So, the external languages worked well with the npm hljs package which is not minified but they didn't work with the packed hljs (which can be downloaded from the hljs website).

I've fixed the issue for the packed version of hljs in the context of this PR: https://github.com/highlightjs/highlight.js/pull/1888. As far as I see the v9.15.1 and newer contains the fix. So, it should work with latest versions of hljs, try to update it in your project.
